app/src/main/java/org/mozilla/focus/screenshot/ScreenshotManager.java

```diff
@@ -85,10 +99,41 @@ private void lazyInitCategories(Context context) {
             if (categories.size() != 0) {
                 return;
             }
-            final JSONObject json;
+            initFromLocal(context);
```

Comment:
maybe we can `initFromRemote` first and then call `initFromLocal` if remote is empty